math/adept2/Portfile

```diff
@@ -0,0 +1,50 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           compilers 1.0
+PortGroup           github 1.0
+
+github.setup        rjhogan Adept-2 752e4b3bd702d56369e4ff10b81ac3e065671e95
+name                adept2
+version             2023.01.13
+revision            0
+categories          math
+license             Apache-2
+maintainers         {@barracuda156 gmail.com:vital.had} openmaintainer
+description         Fast automatic differentiation library in C++
+long_description    Combined array and automatic differentiation library in C++.
+homepage            http://www.met.reading.ac.uk/clouds/adept
```

Comment:
Please prefer `https://` to `http://` for all links wherever possible.

Reply:
Fixed.